front/tests/pages/stdcm/stdcm-page.ts

```diff
@@ -132,10 +134,39 @@ class STDCMPage extends HomePage {
     await this.simulationStatus.waitFor({ timeout: STDCM_SIMULATION_TIMEOUT });
   }
 
+  // Launch the simualtion with mandatory fields empty
+  async launchSimulationWithEmptyFields(): Promise<void> {
+    await this.launchSimulationButton.waitFor();
+    await expect(this.launchSimulationButton).toBeEnabled();
+    await this.launchSimulationButton.click({ force: true });
+    await expect(this.simulationStatus).not.toBeVisible();
+  }
+
```

Comment:
Extract the duplicated logic from launchSimulation() and launchSimulationWithEmptyFields()
```suggestion
async launchSimulation(): Promise<void> {
  await this.launchSimulationButton.waitFor();
  await expect(this.launchSimulationButton).toBeEnabled();
  await this.launchSimulationButton.click({ force: true });
}

async verifyValidSimulationLaunch(): Promise<void> {
  await this.launchSimulation();
  await this.simulationStatus.waitFor({ timeout: STDCM_SIMULATION_TIMEOUT });
}

async verifyInvalidSimulationLaunch(): Promise<void> {
  await this.launchSimulation();
  await expect(this.simulationStatus).not.toBeVisible();
}
```